lua/lualine/extensions/quickfix.lua

```diff
@@ -21,10 +21,21 @@ local M = {}
 function M.init()
```

Comment:
init is a special fuction used to initialize the extension. use a different function to implement this

Reply:
You are right, I have renamed the method to `M.get_colours()`.